api/setup.py

```diff
@@ -57,7 +57,11 @@ def recur_expand(target_root, dir):
                   "examples/subscribe",
                   "examples/unsubscribe",
                   ])] + list(recur_expand('share/zulip', 'integrations/')),
-    scripts=["bin/zulip-send"],
+    entry_points={
+        'console_scripts': [
+            'zulip-send=zulip.send:main',
```

Comment:
@rht: Those commands in `README.md` should still be valid; they don't need to be replaced with `python zulip/send.py`. `zulip/send.py` is turned into an executable interface called `zulip-send` by `console_scripts`. (The format for console_scripts is `name=package.module:mainfunc`) :)